docs/flow/reference/components/prometheus.integration.node_exporter.md

```diff
@@ -0,0 +1,310 @@
+An explicit value in the YAML config takes precedence over the environment
+variable.
+
+
+### `systemd` block
+name | type | description | default | required
+---- | ---- | ----------- | ------- | --------
+enable_restarts | boolean | Enables service unit metric service_restart_total | false | no
+start_time      | boolean | Enables service unit metric unit_start_time_seconds | false | no
+task_metrics    | boolean | Enables service unit tasks metrics unit_tasks_current and unit_tasks_max.
+unit_exclude    | string  | Regexp of systemd units to exclude. Units must both match include and not match exclude to be collected. | `".+\\.(automount\|device\|mount\|scope\|slice)"` | no
+unit_include    | string  | Regexp of systemd units to include. Units must both match include and not match exclude to be collected. | `".+"` | no
+
+### `tapestats` block
+name | type | description | default | required
+---- | ---- | ----------- | ------- | --------
+`ignored_devices` | string | Regexp of tapestats devices to ignore. | `"^$"` | no
+
+### `textfile` block
+name | type | description | default | required
+---- | ---- | ----------- | ------- | --------
+`textfile_directory` | string | Directory to read *.prom files from for the textfile collector. | `""` | no
+
+### `vmstat` block
+name | type | description | default | required
+---- | ---- | ----------- | ------- | --------
+`fields` | string | Regexp of fields to return for the vmstat collector. | `"^(oom_kill\|pgpg\|pswp\|pg.*fault).*"` | no
+
+
+## Exported fields
+The following fields are exported and can be referenced by other components.
+
+Name      | Type                | Description 
+--------- | ------------------- | ----------- 
+`targets` | `list(map(string))` | The targets where the `node_exporter` metrics are exposed to.
+
+For example, the `targets` could either be passed to a `prometheus.relabel`
+component to rewrite the metrics' label set, or to a `prometheus.scrape`
+component that collects the exposed metrics.
+
+## Component health
+
+`prometheus.integrations.node_exporter` is only reported as unhealthy if given
+an invalid configuration. In those cases, exported fields are kept at their
+last healthy values.
+
+## Debug information
+
+`prometheus.relabel` does not expose any component-specific debug information.
+
+## Debug metrics
+
+`prometheus.relabel` does not expose any component-specific debug metrics.
+
+## Collectors List
+The following table lists the available collectors that node_exporter brings
+bundled in. Some collectors may only work on specific operating systems;
+enabling a collector that is not supported by the host OS where Flow is running
+is a no-op.
+
+Users can choose to enable a subset of collectors to limit the amount of
+metrics exposed by the `prometheus.integrations.node_exporter` component,
+or disable collectors that are expensive to run.
+
+| Name             | Description | OS | Enabled by default |
```

Comment:
IMO the collector names here should probably be wrapped in backticks since they're expected to be placed verbatim in the config file.